components/classes/StudentsListPaper.tsx

```diff
@@ -1,264 +1,100 @@
-// Imports
-import { MAXIMUM_EMPTY_COLUMNS, OptionsType } from "@/components/classes/StudentListPrintout";
+import { OptionsType } from "@/components/classes/StudentListPrintout";
+import StudentListTableBody from "@/components/classes/StudentListTableBody";
+import StudentListTableHead from "@/components/classes/StudentListTableHead";
 import PaperPreview from "@/components/common/print/PaperPreview";
 import cn from "@/utils/helpers/cn";
 import getCurrentAcademicYear from "@/utils/helpers/getCurrentAcademicYear";
 import getLocaleName from "@/utils/helpers/getLocaleName";
-import getLocaleString from "@/utils/helpers/getLocaleString";
 import getLocaleYear from "@/utils/helpers/getLocaleYear";
 import { Classroom } from "@/utils/types/classroom";
+import { StylableFC } from "@/utils/types/common";
 import { Student } from "@/utils/types/person";
-import { list } from "radash";
-import { FC } from "react";
 
 /**
  * A preview of the Student List Printout.
  *
- * @param classItem The Class (`ClassWNumber`) to print information of.
- * @param classOverview (`ClassOverview`); used for Class Advisors information.
+ * @param classroom The Classroom to display the Student List for.
  * @param studentList The list of all Students in this Class.
  * @param options Print options.
- *
- * @returns A Paper Preview.
  */
-const StudentsListPaper: FC<{
-  classItem: Pick<Classroom, "id" | "number">;
-  classroomOverview: Pick<
+const StudentsListPaper: StylableFC<{
+  classroom: Pick<
     Classroom,
     "id" | "number" | "class_advisors" | "contacts" | "subjects"
   >;
   studentList: Student[];
   options: OptionsType;
-}> = ({ classItem, classroomOverview, studentList, options }) => {
-  return (
-    <PaperPreview>
-      {/* Header */}
-      <div className="text-center text-[0.925rem] leading-6">
-        <p>
-          {options.language === "en-US"
-            ? "Suankularb Wittayalai School"
-            : "โรงเรียนสวนกุหลาบวิทยาลัย"}
-        </p>
-        <p>
-          {(() => {
-            const grade = Math.floor(classItem.number / 100);

```

Comment:
Why does it have to be divided by 100?

Reply:
`classroom.number` is a number representing the 3-digit number of a classroom. For example, M.604 would be represented as the number `604`.

To get the grade (which is the first digit of the number), we divide the number by 100 and then the decimals by rounding down (`Math.floor`).

See this example:

```ts
classroom.number
// 604

604 / 100
// 6.04

Math.floor(6.04)
// 6
```

Or see a breakdown of how this line is evaluated:

```ts
Math.floor(classroom.number / 100)
Math.floor(604 / 100)
Math.floor(6.04)
6
```